CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
   the metric `tank.sample-too-far-ahead` can be used, this metric counts the data points which
   would be dropped if the enforcement were turned on while it is off.
   #1572 
+* as of v0.13.1-76-g235ecb29 tag support is enabled by default, it can still be disabled though.
+  This means if previously metrics with tags have been ingested while tag support was disabled,
+  then those tags would have been treated as a normal part of the metric name, when tag support
+  now gets enabled due to this change then the tags would be treated as tags.
```

Comment:
it is not clear why this is a breaking change

Reply:
Added some further explanation